Tests/ValetIntegrationTests/SinglePromptSecureEnclaveIntegrationTests.swift

```diff
@@ -159,7 +159,24 @@ class SinglePromptSecureEnclaveIntegrationTests: XCTestCase
             XCTAssertTrue(permutation.canAccessKeychain())
         }
     }
-    
+
+    func test_canAccessKeychain_sharedAppGroup() {
+        guard #available(tvOS 11.0, *) else {
+            return
+        }
```

Comment:
Instead of guarding, can we mark the test as `@available`?

```suggestion
    @available(tvOS 11.0, *)
    func test_canAccessKeychain_sharedAppGroup() {
```

Reply:
unfortunately no. I learned the hard way in earlier Valet PRs that `@available` doesn't work as expected when applied to test methods – all methods are executed, regardless of their availability. I believe this is because XCTest uses the Objective-C runtime to find all selectors that begin with `test`.